health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/MetricChart.js

```
@@ -0,0 +1,154 @@
+            height: 50,
+            whiteSpace: "normal",
+          }}
+        >
+          <span style={{ fontWeight: "500", color: "white" }}>{t(`TIP_${data.name}`)}</span>
+        </span>
+      </div>
+        <span style={{ whiteSpace: "pre" }}>{t("DSS_NO_DATA")}</span>
+    </div>
+    );
+  }
+  let name=t(data?.name)||"";
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'name' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)